run_dir/static/js/running_notes.js

```diff
@@ -147,20 +148,66 @@ function filter_running_notes(search){
         }
     });
 }
+//Count rn's of each category and add badge with number to filter dropdown
+function count_cards(){
+    var cat_cards = {};
+    var all = 0;
+    $('#running_notes_panels').find('.badge').each(function(){
+        var label = $.trim($(this).text());
+        all++
+        if (label){
+            if (label in cat_cards){
+                cat_cards[label]++;
+            }else{
+                cat_cards[label] = 1;
+            }
+        }
+    });
+    $('.btn_count').append('All'+'&nbsp;'+'<span class="badge bg-secondary">'+all+'</span>');
```

Comment:
Could use a bootstrap padding class here instead of `&nbsp;`

Reply:
No, use the bootstrap classes as in my suggestion comment. m or p for margin or padding. Then an optional direction, hyphen then a number between 1 and 5.

eg. `ml-2` for a fairly small left margin. Or `p-5` for lots of padding in all sides. Or `mx-3` for margin on both sides.

Reply:
Aha, I didn't see your comment until now 😄 `mr-2` looks great, I think!